### PR TITLE
Chef Brigade => Sous Chefs

### PR DESCRIPTION
**Pull request by shortdudey123**

### CODE REVIEW

**Comment by shortdudey123** [comment]
Corrected, will merge after travis passes then

File: README.md [hunk 1]
```diff
@@ -450,17 +451,10 @@ https://github.com/spheromak/dhcp-cook
 
 |                      |                                          |
 |:---------------------|:-----------------------------------------|
+| **Maintainer Community** | [Sous Chefs](https://github.com/sous-chefs)
 | **Maintainer**       | [Jacob McCann](https://github.com/jmccann)
 | **Original Author**  | [Jesse Nelson](https://github.com/spheromak)
 | **Author**           | [Matt Ray](https://github.com/mattray)
-| **Contributor**      | [Myles Steinhauser](https://github.com/masteinhauser)
```
Comment (josephholsten) [inline README.md]: Any special motivation for removing the contributors? I'll grant it doesn't add much without a commit history, in which you'd be able to see all the contributors anyway.

File: providers/shared_network.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@
 # Cookbook Name:: dhcp
 # Provider:: dhcp_shared_subnet
 #
-# Copyright:: 2015, Chef Brigade
+# Copyright:: 2015-2016, Sous Chefs
```
Comment (josephholsten) [inline providers/shared_network.rb]: Do we really want copyright headers in every file? If so, we should add them everywhere, elsewise remove them everywhere.

File: README.md (hunk 1 above)
Comment (shortdudey123) [inline README.md]: Generally speaking, we don't list contributors in any of the cookbooks since the commit history has it

**Pull request merged**